connector_easypost_tracker/controllers/main.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from json import loads, dumps
+from openerp import http, SUPERUSER_ID
+from openerp.http import request
+from openerp.addons.connector_easypost.unit.import_synchronizer import (
+    create_connector_session,
+    import_data,
+)
+
+
+class EasypostWebhookController(http.Controller):
+
+    EVENTS = {'tracker.updated': 'easypost.stock.picking.tracking.group'}
+
+    def _get_backend(self, env):
+        return env['easypost.backend'].sudo().search([
+            ('is_default', '=', True)
+        ])
+
+    @http.route([
+        '/api/easypost/webhook',
```

Comment:
As much as I prefer this URL, you don't own this namespace. You own routes under `/connector_easypost_tracker/*`